queries.sql

```diff
@@ -93,3 +93,74 @@ SELECT full_name, COUNT(full_name) AS amount FROM owners
 JOIN animals ON animals.owner_id = owners.id
 GROUP BY full_name
 ORDER BY amount DESC LIMIT 1;
+
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍